src/test/java/jenkins/bouncycastle/FingerPrintTest.java

```diff
@@ -47,11 +48,17 @@
 
     @BeforeClass
     public static void setUpBC() throws URISyntaxException {
-        Security.addProvider(new org.bouncycastle.jce.provider.BouncyCastleProvider());
+        Security.addProvider(new BouncyCastleProvider());
         PEM_FILE = new File(
                 EncodingDecodingTest.class.getClassLoader().getResource("private-key-fingerprint.pem").toURI());
     }
 
+    @AfterClass
```

Comment:
technically not requires as surefire is configured to not reuse forks, but this is good for IDEs